faerie_david/.gitignore

```diff
@@ -1,5 +1,14 @@
-.pio
 .vscode/.browse.c_cpp.db*
 .vscode/c_cpp_properties.json
 .vscode/launch.json
 .vscode/ipch
+
+# For developing rover-Embedded-Lib
+# There has to be a better way of doing this...
+!/.pio
```

Comment:
What the hell is even happening here?

Reply:
I was trying to tell git to track the directories themselves but not anything inside them except for the rover-Embedded-Lib submodule lol
I'm sure there is a better way of doing that, I just didn't know what it was